scripts/constants.py

```diff
@@ -86,7 +86,7 @@
         )
 else:
     NETWORK = {
-        "name": "",
+        "name": "custom_rpc",
```

Comment:
we have already `starknet-devnet` as a name so better not to mix cases. Can you make this `custum-rpc`? and why not adding an env var to possible set it, like `os.getenv("RPC_NAME", "custom-rpc")`?